polaris-react/src/components/Filters/Filters.stories.tsx

```diff
@@ -1289,7 +1301,9 @@ export function WithQueryFieldHidden() {
                 accessibilityLabel={`View details for ${name}`}
               >
                 <h3>
-                  <TextStyle variation="strong">{name}</TextStyle>
+                  <Text variant="bodyMd" fontWeight="bold" as="span">
```

Comment:
Should we simplify the composition here by making the Text component a `h3` tag?

`<Text variant="bodyMd" fontWeight="bold" as="h3">`

Same with other instances in this file.